src/flutter/shell/platform/linux_embedded/window/elinux_window_wayland.h

```diff
@@ -171,6 +171,7 @@ class ELinuxWindowWayland : public ELinuxWindow, public WindowBindingHandler {
   wl_output* wl_output_;
   wl_shm* wl_shm_;
   std::unordered_map<wl_seat*, seat_inputs> seat_inputs_map_;
+  std::unordered_map<uint32_t, wl_seat*> registry_names_to_seat_ptr_;
```

Comment:
Is `registry_names_to_seat_ptr_` really necessary? Because I cannot see any code to intert new elements in the unordered_map.

Reply:
Sorry, I overlooked some code...